client/signup/steps/rewind-form/index.jsx

```diff
@@ -0,0 +1,95 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'gridicons';
+import StepWrapper from 'signup/step-wrapper';
+import Card from 'components/card';
+import CompactCard from 'components/card/compact';
+import CredentialsForm from 'my-sites/site-settings/jetpack-credentials/credentials-form';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+class RewindFormStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	};
+
+	skipStep = () => this.props.goToNextStep();
+
+	onCancel = () => this.props.goToStep( 'rewind-confirm' );
+
+	renderStepContent = () => {
+		const { siteId, translate } = this.props;
+
+		return (
+			<div className="rewind-form__container">
+				<h3 className="rewind-form__title">{ translate( 'Site Credentials' ) }</h3>
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Add site credentials' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).